B9PartSwitch/PartSwitch/PartSwitchFlightDialog.cs

```diff
@@ -104,7 +106,28 @@ private static DialogGUIBase[] CreateOptions(ModuleB9PartSwitch module, IList<Ca
 
             options.Add(new DialogGUIButton(Localization.PartSwitchFlightDialog_CancelString, delegate { } ));
 
-            return options.ToArray();
+            const float buttonHeight = 35;
+            if (buttonHeight * options.Count < 0.75f * Screen.height) return options.ToArray();
+
+            // UnityEngine.TextRenderingModule must be added to project References
+            options.Insert(0, new DialogGUIContentSizer(ContentSizeFitter.FitMode.Unconstrained, ContentSizeFitter.FitMode.PreferredSize));
```

Comment:
If this needs to be first maybe we can just add it before the loop?

Reply:
I think so.  I was a little paranoid about not altering the existing popup behavior in cases where my changes aren't needed.  From a quick test, it doesn't look like having that element causes any changes to the normal popup, so it's probably fine to just always add it first.  I can commit the change if you want.

Reply:
I think it makes sense to do that yeah

Reply:
Ok, done.